content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,417 @@
+---
+title: Pod Eviction
+content_type: concept
+weight: 50
+---
+
+Eviction is the process by which {{<glossary_tooltip text="pods" term_id="pod" length="short">}}
+in a {{<glossary_tooltip text="node" term_id="node" length="short">}} are
+terminated. 
+
+In Kubernetes, there are two types of pod eviction:
+
+  1. Graceful eviction
+  2. Out-of-resource eviction ({{<glossary_tooltip text="kubelet" term_id="kubelet" length="short">}}
+     eviction)
+
+{{<caution>}}
+Out-of-resource eviction does not share any characteristics
+with graceful eviction. Do not use the two eviction types interchangeably. 
```

Comment:
You don't "use" out-of-resource eviction. The system applies it.

Reply:
I'll remove that sentence. Maybe "Out-of-resource eviction and graceful eviction are not interchangeable"?

Reply:
I think the caution is unnecessary.